pkgs/by-name/fa/famistudio-bin/package.nix

```
@@ -0,0 +1,70 @@
+{ lib
+, stdenv
+, fetchzip
+, autoPatchelfHook
+, dotnet-runtime
+, ffmpeg
+, libglvnd
+, makeWrapper
+, openal
+}:
+
+stdenv.mkDerivation (finalAttrs: {
+  pname = "famistudio-bin";
+  version = "4.1.3";
+
+  src = fetchzip {
+    url = "https://github.com/BleuBleu/FamiStudio/releases/download/${finalAttrs.version}/FamiStudio${lib.strings.concatStrings (lib.splitVersion finalAttrs.version)}-LinuxAMD64.zip";
```

```suggestion
    url = "https://github.com/BleuBleu/FamiStudio/releases/download/${finalAttrs.version}/FamiStudio${lib.replaceStrings ["."] [""] finalAttrs.version}-LinuxAMD64.zip";
```
Using replaceStrings is simpler and the internal placings does not matter for outside uses, please don't use them.